Makefile

```diff
@@ -15,3 +17,9 @@ tidy:
 e2e:
 	test/e2e.sh
 
+.PHONY: golangci-lint
+golangci-lint: $(GOLANGCILINT)
+	$(GOPATH)/bin/golangci-lint run --verbose
+
+$(GOLANGCILINT):
```

Comment:
how do you run this rule? I haven't seen this makefile convention before

Reply:
`make golangci-lint`